tier1/interface.cpp

```diff
@@ -401,22 +401,6 @@ CSysModule *Sys_LoadModule( const char *pModuleName )
 	// prior to the call to this routine.
 	HMODULE hDLL = 0;
 
-	char alteredFilename[ MAX_PATH ];
-	if ( IsPS3() )
```

Comment:
Please replace this with `IsPlatformPS3` instead of removing it. I want to avoid unnecessarily big changes, because it makes it harder to synchronize code.